### ISSUE TITLE
Error: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by nio-sfn** [anchordotdev/cli#9]
## Environment

- Model: Apple M1 Pro
- macOS: 14.4.1 (23E224)

## brew info

```
==> anchordotdev/tap/anchor: stable 0.0.26
Command-line tools for Anchor.dev
https://anchor.dev/
Not installed
From: https://github.com/anchordotdev/homebrew-tap/blob/HEAD/Formula/anchor.rb
License: MIT
```

---

After running the `anchor auth signin` command, and the browser finishes authenticating, I get the following error:

```
Caught panic:

runtime error: invalid memory address or nil pointer dereference

Restoring terminal...

goroutine 33 [running]:
runtime/debug.Stack()
	/Users/runner/hostedtoolcache/go/1.21.9/arm64/src/runtime/debug/stack.go:24 +0x64
runtime/debug.PrintStack()
	/Users/runner/hostedtoolcache/go/1.21.9/arm64/src/runtime/debug/stack.go:16 +0x1c
github.com/charmbracelet/bubbletea.(*Program).Run.func1()
	/Users/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.25.0/tea.go:478 +0x88
panic({0x100692040?, 0x100b48010?})
	/Users/runner/hostedtoolcache/go/1.21.9/arm64/src/runtime/panic.go:920 +0x26c
github.com/anchordotdev/cli/ui.(*Driver).View(0x140003000b0)
	/Users/runner/work/cli/cli/ui/driver.go:176 +0x148
github.com/charmbracelet/bubbletea.(*Program).eventLoop(0x14000344a90, {0x1007475b8?, 0x140003000b0?}, 0x0?)
	/Users/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.25.0/tea.go:413 +0x6e4
github.com/charmbracelet/bubbletea.(*Program).Run(0x14000344a90)
	/Users/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.25.0/tea.go:543 +0x6d8
github.com/anchordotdev/cli.NewCmd[...].func1.1.1()
	/Users/runner/work/cli/cli/cmd.go:274 +0x64
created by github.com/anchordotdev/cli.NewCmd[...].func1.1 in goroutine 1
	/Users/runner/work/cli/cli/cmd.go:271 +0x1f4
```

**Comment by geemus** [comment]
Thanks for the detailed report. I was able to replicate it as well.

At least for me, the authentication did succeed before the error. So I was able to run other commands, so you likely could proceed if you want to, even before I'm able to fix the bug.

I'll take a look in the morning to determine how best to get this fixed.

**Comment by geemus** [comment]
After a bit more inspection it turned out to be a pretty easy fix which we applied elsewhere but somehow missed this one. Sorry for the trouble, it should now be fixed in the newly released v0.0.27, which you should be able to get by running `brew update && brew upgrade anchor`. Just let us know if you have any more issues or have feedback to share, and thanks again for the detailed report.
